bot/cogs/cogs.py

```diff
@@ -183,7 +185,8 @@ def __init__(self, bot: AutoShardedBot):
         if full_cog:
             if full_cog == "*":
                 all_cogs = [
-                    f"bot.cogs.{fn[:-3]}" for fn in os.listdir("bot/cogs")
+                    f"bot.cogs.{fn[:-3]}"
+                    for fn in os.listdir("bot/cogs")
                     if os.path.isfile(f"bot/cogs/{fn}") and fn.endswith(".py") and "_" not in fn
```

Comment:
I mean, if we're gonna do this then this line should be split into three lines too.